todo_app/data/trello_items.py

```diff
@@ -0,0 +1,194 @@
+import requests
+from flask import current_app as app
+import dateutil.parser
+from datetime import datetime
+
+def get_auth_params():
+    return { 'key': app.config['API_KEY'], 'token': app.config['API_TOKEN'] }
+
+def build_url(endpoint):
+    return app.config['BASE_URL'] + endpoint
+
+def build_params(params = {}):
```

Comment:
Style point: private methods in Python that aren't intended to be used outside of the current file normally have their name prefixed with `_`, e.g. `_build_params`.

Reply:
Thanks Ben. I have updated to reference double underscore for the private mthods.